discovery-provider/es-indexer/src/indexNames.ts

```diff
@@ -1,5 +1,5 @@
 export const indexNames = {
-  playlists: 'playlists17',
+  playlists: 'playlists18',
```

Comment:
updating playlists es indexer so that es re-indexes playlists with their latest playlist routes created from the backfill